pkg/ring/lifecycler.go

```diff
@@ -63,6 +70,7 @@ func (cfg *LifecyclerConfig) RegisterFlagsWithPrefix(prefix string, f *flag.Flag
 	}
 
 	f.IntVar(&cfg.NumTokens, prefix+"num-tokens", 128, "Number of tokens for each ingester.")
+	f.StringVar(&cfg.TokensGeneratorStrategy, prefix+"tokens-generator-strategy", randomTokenStrategy, fmt.Sprintf("Algorithm used to generate new tokens. Supported Values: Random, %s", strings.Join(supportedTokenStrategy, ",")))
```

Comment:
Let's remove `Random,`? I think it is already mentioned in `supportedTokenStrategy`

Reply:
done!